stringcalculator/stringcalculator/CalculatorAddShould.cs

```diff
@@ -29,5 +29,17 @@ public void ReturnOneGivenNumeralOne()
             // Assert
             Assert.AreEqual(1, result);
         }
+        [Test]
```

Comment:
Let's combine this test with the previous one, using `[TestCase]`. You can change the name of the test to something like `ShouldReturnIntegerGivenOneInteger` and have the test take in parameters `string input, int expectedResult`. Update the test code to pass the `input` to the `Add` method and change the assert to expect `expectedResult`. Then add two attributes to the test:
   [TestCase("1",1)]
   [TestCase("2",2)]

See if that works and if the tests still pass. You can add additional test cases if you think they're helpful.